sjb/config/test_cases/azure_build_node_image_rhel_310.yml

```diff
@@ -31,6 +34,13 @@ actions:
   - type: "host_script"
     title: "move secrets to remote host"
     script: |-
+      set +x
+      export KUBECONFIG=/var/lib/jenkins/.azure/kubeconfig
+      NEW_TOKEN=$(oc get secret aws-reg-token --template={{.data.token}} | base64 -d)
+      if [[ -n $NEW_TOKEN ]]; then
+        echo $NEW_TOKEN > /var/lib/jenkins/.azure/registry
```

Comment:
don't write to the jenkins filesystem, it introduces a (tiny) race - echo the token up to the box via ssh echo.  Think about the filename you choose - make it descriptive